src/cmr/concepts/orderOption.js

```diff
@@ -0,0 +1,42 @@
+import Concept from './concept'
+
+export default class OrderOption extends Concept {
+  /**
+   * Instantiates an OrderOption object
+   * @param {Object} headers HTTP headers provided by the query
+   * @param {Object} requestInfo Parsed data pertaining to the Graph query
+   * @param {Object} params GraphQL query parameters
+   */
+  constructor(headers, requestInfo, params) {
+    // This concept uses the "-" character to delineate spaces in CMR we must pass it
+    // in this form to fetch order option concepts from CMR
+    super('order-options', headers, requestInfo, params)
+  }
+
+  /**
+   * Parse and return the array of data from the nested response body
+   * @param {Object} jsonResponse HTTP response from the CMR endpoint
+   */
+  parseJsonBody(jsonResponse) {
+    const { data } = jsonResponse
+
+    const { items } = data
```

Comment:
We should have some null checks for safety.

Reply:
Yea I'm taking a closer look at this one. This is overriding the parent classes method so `orderOption.js `(the concept definition this file is in) doesn't have an accompanying test file since it is largely just overriding the concept class the same is true of all concepts in graphql, I imagine that is why none of these have a default null check case since that would require making a new test file for each of the concepts to get test coverage on this possible branch . The `parseUmmBody` method in the concept class is equivalent to this overridden `parseJsonBody` to handle cmr responses. This method only gets called once in the `parseJson` method from the concept class which is in turn called in another method which has a guard around the null case and passes the response down to here
```
 if (jsonResponse) {
        await this.parseJson(jsonResponse, jsonKeys)
      }
```
which is being covered by tests. Do you think I should add tests for all the concepts for the null case?